### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by takano32**
Updated some steps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -255,28 +267,50 @@ $ mysql -uairone -h127.0.0.1 -ppassword -e 'create database airone'
...
+$ source .venv/bin/activate
+$ celery -A airone worker -l info
+```
+
+## Setup Static files
+
+Use [WhiteNose](http://whitenoise.evans.io/) for serving static files.
+
+
+
```
Comment (userlocalhost) [inline README.md]: It's cooler to remove unnecessary blank lines, I think.

**Review by userlocalhost — COMMENTED** [review]
Thank you for updating README 👍

File: README.md (hunk 1 above)
Comment (takano32) [reply README.md]: I removed.

**Review by userlocalhost — APPROVED** [review]
Thank you very much for fixing! 👍

**Pull request merged**